lib/spec/simpleofflinecache.sw-spec.js

```diff
@@ -118,19 +105,21 @@ describe('SimpleOfflineCache instance', function () {
     var request = new Request('/index.html?param=zzz');
 
     var cacheObj = {
-      match: sinon.stub().returns(Promise.resolve(null)),
-      put: sinon.stub().returns(Promise.resolve())
+      match: sinon.stub().returns(Promise.resolve({}))
     };
-    sinon.stub(self.cacheHelper, 'getCache').returns(
+    // Clean the cached request
+    delete offline.cacheRequest;
+    sinon.stub(caches, 'open').returns(
       Promise.resolve(cacheObj));
     offline.onFetch(request).then(function() {
-      expect(self.cacheHelper.getCache.calledOnce).to.be.true;
+      expect(caches.open.calledOnce).to.be.true;
```

Comment:
As stated in the [last general comment](https://github.com/gaia-components/serviceworkerware/pull/29#issuecomment-119858117) you should separate the test of caching the cache request in a different thread. Notice there you should call `onFetch()` and assert `caches.open` is called once.